pkgs/servers/openvscode-server/default.nix

```diff
@@ -164,6 +164,10 @@ in stdenv.mkDerivation rec {
     ln -s ${nodejs}/bin/node $out
   '';
 
+  passthru.tests = {
+    inherit (nixosTests) openvscode-server;
+  };
```

Comment:
This file is missing `nixosTests` in lines 1-4.
Please add it :)

```patch
diff --git a/pkgs/servers/openvscode-server/default.nix b/pkgs/servers/openvscode-server/default.nix
index f0133633fcd..f5c17fcaff6 100644
--- a/pkgs/servers/openvscode-server/default.nix
+++ b/pkgs/servers/openvscode-server/default.nix
@@ -1,7 +1,7 @@
 { lib, stdenv, fetchFromGitHub, buildGoModule, makeWrapper
 , cacert, moreutils, jq, git, pkg-config, yarn, python3
 , esbuild, nodejs_16, libsecret, xorg, ripgrep
-, AppKit, Cocoa, Security, cctools }:
+, AppKit, Cocoa, Security, cctools, nixosTests }:
 
 let
   system = stdenv.hostPlatform.system;
```

Reply:
Argh ! Just did.